crawl-ref/source/dat/strings/de/messages.txt

```diff
@@ -9945,7 +9945,7 @@ The vine drags %s backwards!
 Die Rebe zieht {acc}%s nach hinten!
 %%%%
 The tentacle pulls %s backwards!
-Der Tentakel zieht {acc}%s nach hinten!
+Das Tentakel zieht {acc}%s nach hinten!
```

Comment:
This should be:
Das Tentakel zieht sich zurück!

Reply:
It's pulling another monster though (so not sich).
Should it be: Das Tentakel zieht {acc}%s zurück!

Reply:
OK. Thanks